libraries/AP_InertialSensor/AP_InertialSensor_Backend.h

```diff
@@ -25,12 +25,25 @@
 #include <inttypes.h>
 
 #include <AP_Math/AP_Math.h>
+#include "AP_Math/fourier.h"
 
 #include "AP_InertialSensor.h"
 
+
+const int SAMPLE_RATE=1600;
+const int MIN_FREQ=1;
+const int BUF_SIZE=1600;//SAMPLE_RATE/MIN_FREQ;
```

Comment:
if this 1600 value is SAMPLE_RATE/MIN_FREQ, they why not just use BUF_SIZE = SAMPLE_RATE/MIN_FREQ;

Reply:
ok fixed